src/ChatFeed/MessageForm/AttachmentInput/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import { Properties } from 'csstype';
+
+import { Props } from './props';
+import { styles } from './styles';
+
+export const AttachmentInput: React.FC<Props> = ({
+  onSelectFiles,
+  style = {},
+}: Props) => {
+  const [hovered, setHovered] = useState<boolean>(false);
+  const hoverStyle: Properties = { color: hovered ? '#06c' : '#444' };
+
+  const onSelect = (e: React.ChangeEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    const target = e.target as HTMLInputElement;
+    const files = target.files;
+    onSelectFiles && onSelectFiles(files);
+  };
+
+  return (
+    <form
+      className="uploader"
```

Comment:
this className doesn't follow the `ce-<component> pattern`

Reply:
Yep - good point